runtime/doc/eval.txt

```diff
@@ -1552,6 +1552,19 @@ v:event		Dictionary of event data for the current |autocommand|.  Valid
 					operation.
 			regtype		Type of register as returned by
 					|getregtype()|.
+			completed_item    Current selected complete item on
+					|MenuPopupChanged|, Is `{}` when no complete
+					item selected.
+			height 		Height of popup menu on |MenuPopupChanged|
+			width   	width of popup menu on |MenuPopupChanged|
+			row  	 	Row count of popup menu on |MenuPopupChanged|,
+					relative to screen.
+			col  	 	Col count of popup menu on |MenuPopupChanged|,
+					relative to screen.
+			size 		Total size of complete items on
+					|MenuPopupChanged|.
+			scrollbar 	Scrollbar exists for popup menu, is `1` when
```

Comment:
Sounds like this should be `v:true` and `v:false`. Or is the point to preanticipate scrollbar with width larger than one (which I find unlikely)?